app/server.js

```diff
@@ -0,0 +1,71 @@
+const ttn = require("ttn");
+const app = require("express")();
+const http = require("http").Server(app);
+const bodyParser = require("body-parser");
+
+const googleMapsClient = require("@google/maps").createClient({
+  key: "xxxxxxxxxxxxxxxxxxxxxx"
+});
+
+var appID = "xxxxxxxxxxxxx";
+var accessKey = "ttn-account-v2.xxxxxxxxxxxxxxxxxxxxxx";
+
+var address = "";
+var wifibssids = [];
+
+var setAddresses = false;
```

Comment:
What is it for?

Reply:
This idea was to read the addresses sent by the device only once, on the first message, but I can make it as it was before: updating the addresses after each uplink message

Reply:
Yes please